sonar-ws/src/main/protobuf/ws-commons.proto

```diff
@@ -72,17 +72,6 @@ enum RuleStatus {
   REMOVED = 3;
 }
 
-message User {
```

Comment:
This was only used for issues : it was not common. If tomorrow we need a common user used by different domain we'll create it